modules/swagger-codegen/src/main/resources/lua/luarocks.mustache

```diff
@@ -0,0 +1,28 @@
+package = "{{{packageName}}}"
+version = "{{{packageVersion}}}"
+source = {
+	url = "https://github.com/{{#gitUserId}}{{.}}{{/gitUserId}}{{^gitUserId}}YOUR_GIT_USERNAME{{/gitUserId}}/{{#gitRepoId}}{{.}}{{/gitRepoId}}{{^gitRepoId}}YOUR_GIT_REPO{{/gitRepoId}}.git"
+}
+
+description = {
+	summary = "API client genreated by Swagger Codegen",
+	detailed = [[
+{{{appDescription}}}]],
+	homepage = "https://github.com/swagger-api/swagger-codegen",
+	license = "Unlicense",
+	maintainer = "Swagger Codegen contributors",
+}
+
+dependencies = {
+	"lua >= 5.2"
+}
+
+build = {
+	type = "builtin",
+	modules = {
+		["http.request"] = "http/request.lua";
```

Comment:
This section is meant to list the files you install. not the dependencies you need